client/app/configuration/em-services/ServicesController.js

```diff
@@ -25,7 +25,7 @@ angular.module('EnvironmentManager.configuration').controller('ServicesControlle
       var cluster = $routeParams.cluster;
       var service = $routeParams.service;
 
-      vm.selectedOwningCluster = cluster || SHOW_ALL_OPTION;
+      vm.selectedOwningCluster = cluster || localstorageservice.getValueOrDefault('em-selections-team', SHOW_ALL_OPTION);
```

Comment:
This constant should be common, 'em-selections-team' is present in many places ...

Reply:
Valid point! I will move this across to a constants factory for these values. This will assist in further localstorage use too. ☺️